rust/blockstore/src/arrow/ordered_blockfile_writer.rs

```diff
@@ -200,6 +200,7 @@ impl ArrowOrderedBlockfileWriter {
             blocks,
             self.root,
             self.id,
+            None, // no op
```

Comment:
why?

Reply:
I was under the impression that `ArrowOrderedBLockfileWriter` should be a no op but I think that's a misunderstanding. Will update